learing_journal/scripts/initializedb.py

```diff
@@ -39,9 +39,9 @@ def main(argv=sys.argv):
     # The below lines populates the database initialy
 
     session_factory = get_session_factory(engine)
-
-    with transaction.manager:
-        dbsession = get_tm_session(session_factory, transaction.manager)
-        for entry in ENTRIES:
-            row = MyModel(title=entry['title'], body=entry['body'], date=entry['date'], id=entry['id'])
-            dbsession.add(row)
+    # 
+    # with transaction.manager:
+    #     dbsession = get_tm_session(session_factory, transaction.manager)
+    #     for entry in ENTRIES:
+    #         row = MyModel(title=entry['title'], body=entry['body'], date=entry['date'], id=entry['id'])
+    #         dbsession.add(row)
```

Comment:
remove dead code.  No need to keep corpses laying around in your committed code.